docs/sources/community/maintaining/_index.md

```diff
@@ -2,9 +2,8 @@
 title: Maintaining
 description: Grafana Loki Maintainers' Guide
 aliases: 
-- /docs/loki/latest/maintaining
-- /docs/loki/latest/community/maintaining
-weight: 1200
+- ../../maintaining/
```

Comment:
I believe the intention is to redirect the page /docs/loki/latest/maintaining/, so the alias is: 
```suggestion
- ../maintaining/
```